advanced_functionality/multi_model_linear_learner_home_value/linear_learner_multi_model_endpoint_inf_pipeline.ipynb

```diff
@@ -0,0 +1,2859 @@
+{
```

Comment:
<p>I think we should show how customers can access these metrics for their endpoint they created in this notebook (in the CloudWatch console in the /aws/sagemaker/Endpoints namespace)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Reply:
The namespaces are clear in the graph attached.